### PR TITLE
Fix encoding of shadowing activities

### PR DESCRIPTION
**Pull request by yux0**
1. Fix shadower worker accessibility
2. Fix shadowing activity encoding with golang
3. Minor fixes in shadower worker

### CODE REVIEW

File: src/main/java/com/uber/cadence/internal/shadowing/ReplayWorkflowActivityImpl.java [hunk 1]
```diff
@@ -114,21 +111,22 @@ public ReplayWorkflowActivityResult replay(ReplayWorkflowActivityParams request)
       successCount += oneReplayResult.getSucceeded();
       failedCount += oneReplayResult.getFailed();
       skippedCount += oneReplayResult.getSkipped();
-      ReplayWorkflowActivityResult heartbeatResult =
-          new ReplayWorkflowActivityResult()
-              .setSucceeded(successCount)
-              .setFailed(failedCount)
-              .setSkipped(skippedCount);
+      ReplayWorkflowActivityResult heartbeatResult = new ReplayWorkflowActivityResult();
```
Comment (yycptt) [inline src/main/java/com/uber/cadence/internal/shadowing/ReplayWorkflowActivityImpl.java:114]: nit: looks like we can reuse `heartbeatResult` defined above? no strong opinion on this, feel free to ignore.

File: src/main/java/com/uber/cadence/internal/shadowing/ReplayWorkflowActivityImpl.java [hunk 2]
```diff
@@ -114,21 +111,22 @@ public ReplayWorkflowActivityResult replay(ReplayWorkflowActivityParams request)
...
+      heartbeatResult.setSucceeded(successCount);
+      heartbeatResult.setFailed(failedCount);
+      heartbeatResult.setSkipped(skippedCount);
       Activity.heartbeat(new HeartbeatDetail(heartbeatResult, replayIndex));
     }
-    return new ReplayWorkflowActivityResult()
-        .setSucceeded(successCount)
-        .setFailed(failedCount)
-        .setSkipped(skippedCount);
+    ReplayWorkflowActivityResult result = new ReplayWorkflowActivityResult();
```
Comment (yycptt) [inline src/main/java/com/uber/cadence/internal/shadowing/ReplayWorkflowActivityImpl.java:120]: nit: same here, looks like `heartbeatResult` can be reused.

File: src/main/java/com/uber/cadence/testing/WorkflowShadower.java [hunk 3]
```diff
@@ -34,6 +34,7 @@
 
 public final class WorkflowShadower {
   private static final long SLEEP_INTERVAL = 300L;
+  private static final int PAGE_SIZE = 1000;
```
Comment (yycptt) [inline src/main/java/com/uber/cadence/testing/WorkflowShadower.java:37]: I think server has a default value to this?

File: src/main/java/com/uber/cadence/worker/ShadowingWorker.java [hunk 4]
```diff
@@ -81,31 +88,31 @@
             .build();
     activityWorker =
         new SyncActivityWorker(
-            client.getService(), client.getOptions().getDomain(), this.taskList, activityOptions);
+            client.getService(), shadowerConstants.LocalDomainName, this.taskList, activityOptions);
```
Comment (yycptt) [inline src/main/java/com/uber/cadence/worker/ShadowingWorker.java:91]: 👍

**Review by yycptt — APPROVED** [review]

File: src/main/java/com/uber/cadence/internal/shadowing/ReplayWorkflowActivityImpl.java (hunk 1 above)
Comment (yux0) [reply src/main/java/com/uber/cadence/internal/shadowing/ReplayWorkflowActivityImpl.java]: updated

File: src/main/java/com/uber/cadence/internal/shadowing/ReplayWorkflowActivityImpl.java (hunk 2 above)
Comment (yux0) [reply src/main/java/com/uber/cadence/internal/shadowing/ReplayWorkflowActivityImpl.java:121]: updated

File: src/main/java/com/uber/cadence/testing/WorkflowShadower.java (hunk 3 above)
Comment (yux0) [reply src/main/java/com/uber/cadence/testing/WorkflowShadower.java]: removed

**Pull request merged**